sci-libs/vtk/vtk-9.3.1.ebuild

```diff
@@ -41,19 +41,21 @@ SLOT="0/${MY_PV}"
 KEYWORDS="~amd64 ~arm ~arm64 ~x86 ~amd64-linux ~x86-linux"
 
 # TODO: Like to simplifiy these. Mostly the flags related to Groups.
-IUSE="all-modules boost cuda debug doc examples ffmpeg +freetype gdal gles2-only imaging
-	java las +logging mpi mysql odbc opencascade openmp openvdb pdal postgres python qt5
-	qt6 +rendering sdl tbb test +threads tk video_cards_nvidia +views vtkm web"
+IUSE="all-modules boost +cgns cuda debug doc examples ffmpeg +freetype gdal gles2-only imaging
+	java las +logging minimal mpi mysql +netcdf odbc opencascade openmp openvdb pdal postgres
+	python qt5 qt6 +rendering sdl tbb test +threads tk video_cards_nvidia +views vtkm web"
```

Comment:
We should consider dropping `qt5` in this release, the latest freecad rc works reasonably well with qt6 and as I remember that was the only reverse dependency.

In any case, we should definitely drop this auto-magic behaviour:
```
use qt6 && use qt5 && ewarn "Both qt5 and qt6 USE flags have been selected. Using qt5!"
```
so we can get rid of the weird `sci-libs/vtk[qt6,-qt5]` dependency in the freecad ebuild.